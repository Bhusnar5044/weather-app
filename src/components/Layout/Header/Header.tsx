import * as React from 'react';
import { Link } from 'react-router-dom';

export const Header: React.FC = () => {
    return (
        <div className="h-18 bg-gray-800">
            <div className="flex items-center p-4 ml-10 w-l">
                <Link to="/" className="mr-2">
                    WEATHER-APP
                </Link>
            </div>
        </div>
    );
};
