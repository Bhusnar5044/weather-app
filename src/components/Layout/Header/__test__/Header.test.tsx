import { renderWithRouter } from '@test-utils';
import { Header } from '../Header';

describe('Header', () => {
    it('should render properly', () => {
        const { container } = renderWithRouter(<Header />);
        expect(container).toMatchSnapshot();
    });
});
