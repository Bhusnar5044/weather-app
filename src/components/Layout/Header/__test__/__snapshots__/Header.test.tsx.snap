// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
<div>
  <div
    class="h-18 bg-gray-800"
  >
    <div
      class="flex items-center p-4 ml-10 w-l"
    >
      <a
        class="mr-2"
        href="/"
      >
        WEATHER-APP
      </a>
    </div>
  </div>
</div>
`;
